arm-resources/resources/2016-02-01/swagger/resources.json

```diff
@@ -1877,6 +1877,22 @@
         "properties": {
           "type": "object",
           "description": "Gets or sets the resource properties."
+        },
+        "kind": {
+          "type": "string",
+          "description": "Gets or sets the kind of the resource."
+        },
+        "managedBy": {
+          "type": "string",
+          "description": "Gets or sets the managedBy property of the resource."
+        },
+        "sku": {
+          "$ref": "#/definitions/Sku",
+          "description": "Gets or sets the plan of the resource."
+        },
+        "identity": {
+          "$ref": "#/definitions/Identity",
+          "description": "Gets or sets the identity of the resource."
```

Comment:
As in "Identity for the resource"? This doesn't conform to the existing practice we follow for other definitions, like Plan